spec/requests/api/posts_spec.rb

```diff
@@ -364,7 +364,7 @@
       produces 'application/json'
       response '200', 'post updated' do
         schema type: :object, properties: {
-          locked: { type: :string },
+          locked: { type: :boolean },
```

Comment:
I thought you said it has been working at least one in build, how does it explain the randomness?

Reply:
Actually I'm wrong... the build didn't run at all

![Screenshot from 2022-06-06 16-07-12](https://user-images.githubusercontent.com/4335742/172121372-65a150b2-564e-433b-bec3-b4be949be38b.png)
 in https://github.com/discourse/discourse/pull/16997

Reply:
Makes sense now :D Thanks.